src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -106,6 +125,18 @@ public void setDescription(String description) throws IOException {
         root.createRequest().method("PATCH").with("description", description).withUrlPath(api("")).send();
     }
 
+    /**
+     * Sets privacy.
+     *
+     * @param privacy
+     *            the privacy
+     * @throws IOException
+     *             the io exception
+     */
+    public void setPrivacy(Privacy privacy) throws IOException {
+        root.createRequest().method("PATCH").with("privacy", privacy).withUrlPath(api("")).send();
```

Comment:
@timja 
This matches the existing behavior of other set methods on this class. There have been issues filed describing this is not great.  

This is not a blocker for this PR, but I'd like your thoughts on #589.

Reply:
agree, changed to update.
I didn't like the name but I followed the other method for consistency in the class

Reply:
Glad you agree. But leave this for now. I don't want to do this unevenly (some update some set).  😄